vite-7/README.md

```
@@ -0,0 +1,59 @@
+
+## Build
+
+```bash
+npm build
+```
+
+Output is generated in the `dist/` directory.
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)